### ISSUE TITLE
Desktop search button doesn't submit the form

### ISSUE THREAD

**Issue by jonlow** [thememphisagency/fcbresponsive#37]
The desktop search button is just an anchor tag that doesn't link to anything. Make sure a click submits this form.

Ideally this should be a input or button for accessibility, but get this working using JS first due to time constraints

**Comment by yorky001** [comment]
Fixed but just needs browser checking. It's now a button element.

**Comment by matthewattanasio** [comment]
Doesnt Work in Firefox Windows....
